### DISCUSSION TITLE
How to control the LAN access for each client?

### DISCUSSION THREAD

**Discussion by Jefffish09** [wg-easy/wg-easy#579] [Q&A] [answered]
Hi,

I have no idea that if I set up three clients in wg-easy, such client A, client B and client C.

Now, I want the client A and client C can access both my LAN network and Internet while client B only can access Internet.

How can I achieve this?

**Comment by thomdall** [comment]
want to do the same, i just want a WAN only wireguard service but not sure how to do that with iptable and stuff.

**Comment by kaaax0815** [comment, marked as answer]
You would have to write your own iptable rule for that.
There will be documentation for such use cases (noted here https://github.com/wg-easy/wg-easy/discussions/1179#discussioncomment-10264297)

Sometimes in the future maybe its possible to configure that easily through the UI

**Reply by Jefffish09** [reply]
Great! Thanks for your reply. Configuring in UI is easier for beginners.

**Reply by fahadshery** [reply]
> Great! Thanks for your reply. Configuring in UI is easier for beginners.

hi,

I wish to do the same. were you able to do it? if yes then could you share your conf file pls?

**Reply by Jefffish09** [reply]
> > Great! Thanks for your reply. Configuring in UI is easier for beginners.
> 
> hi,
> 
> I wish to do the same. were you able to do it? if yes then could you share your conf file pls?

Sorry, I haven't worked it out yet.

**Reply by thomdall** [reply]
`WG_POST_UP=iptables -I FORWARD -i wg0 -d 10.0.0.0/24 -j REJECT; iptables -I FORWARD -i wg0 -s 10.12.0.2,10.12.0.13 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE`
`WG_POST_DOWN=iptables -I FORWARD -D wg0 -d 10.0.0.0/24 -j REJECT; iptables -I FORWARD -D wg0 -s 10.12.0.2,10.12.0.13 -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE`

I figured out, two environment variables, see above.
this is what i use, 10.0.0.0/24 is my lan, 10.12.0.2 and 10.12.0.13 are two peers that have access to my lan.
Just swap out those address with your own and test
